package.json

```diff
@@ -4,10 +4,16 @@
   "license": "EPL",
   "scripts": {
     "ng": "node_modules/@angular/cli/bin/ng",
+    "upgradeTesteditorOrgDependencies": "yarn upgrade @testeditor",
+    "prestart": "npm run upgradeTesteditorOrgDependencies",
```

Comment:
just for consistency (shouldn't change the actual execution or anything), should we use `yarn run…` throughout?

_Edit:_ there's also `$npm_execpath`, which should return the path to the package manager's executable, i.e. the one that is currently running.